delete.go

```diff
@@ -2,19 +2,11 @@ package buildah
 
 import (
 	"fmt"
-	"os"
 )
 
 // Delete removes the working container.  The Builder object should not be used
```

Comment:
Yes, though I wouldn't be opposed to changing the comment to reference it as a buildah.Builder object.